app/gui/view/graph-editor/src/component/node/input/widget/list_editor.rs

```diff
@@ -226,6 +234,7 @@ impl Widget {
 struct Model {
     self_id:              WidgetIdentity,
     list:                 ListEditor<ListItem>,
+    append_insert:        Option<object::Instance>,
```

Comment:
It's the append insertion point element. We need to store it, because its margin is modified in the FRP network. I'm going to rename it to `append_insertion_point`.

Reply:
Even with the new name, a comment would be useful, as someone could be not accustomed to span-tree terminology.